delivery.yaml

```diff
@@ -0,0 +1,42 @@
+build_steps:
```

Comment:
I think the commands here are quite specific to the build system e.g base Ubuntu 16.04, so adding it to Makefiles would not be so helpful IMO. I think it's better to contain it all in the context of the delivery.yaml.